parm/parm_fv3diag/diag_table_da

```diff
@@ -10,8 +10,8 @@
 "gfs_dyn",     "snowwat",     "snmr",         "fv3_history",    "all",  .false.,  "none",  2
 "gfs_dyn",     "rainwat",     "rwmr",         "fv3_history",    "all",  .false.,  "none",  2
 "gfs_dyn",     "graupel",     "grle",         "fv3_history",    "all",  .false.,  "none",  2
-#"gfs_dyn",     "ice_nc",      "nccice",       "fv3_history",    "all",  .false.,  "none",  2
-#"gfs_dyn",     "rain_nc",     "nconrd",       "fv3_history",    "all",  .false.,  "none",  2
+"gfs_dyn",     "ice_nc",      "nccice",       "fv3_history",    "all",  .false.,  "none",  2
```

Comment:
See above.

Reply:
This change is in sync with using Thompson MP @CatherineThomas-NOAA is aware and requested I included this change with the other related changes.